packages/financial-templates-lib/src/logger/PagerDutyTransport.ts

```diff
@@ -45,8 +46,12 @@ export class PagerDutyTransport extends Transport {
   // Note: info must be any because that's what the base class uses.
   async log(info: any, callback: (error?: unknown) => void): Promise<void> {
     try {
-      // If the message has markdown then add it and the bot-identifier field. Else put the whole info object as a string
-      const logMessage = info.mrkdwn ? info.mrkdwn + `\n${info["bot-identifier"]}` : JSON.stringify(info);
+      // If the message has markdown then add it (with removed anchor text from links that is not supported by PD) and
+      // the bot-identifier field. Else put the whole info object as a string.
+      const logMessage =
+        typeof info.mrkdwn === "string"
+          ? removeAnchorTextFromLinks(info.mrkdwn) + `\n${info["bot-identifier"]}`
```

Comment:
Sorry, why do we need the newline + bot identifier field? Wouldn't that already be in the log somwhere?

Reply:
We already had this before. I expect PDv1 did not have another way to detect the source